pkg/trace/config/config_test.go

```diff
@@ -41,7 +41,9 @@ func TestConfigHostname(t *testing.T) {
 			fallbackHostnameFunc = os.Hostname
 		}()
 		_, err := Load("./testdata/site_override.yaml")
-		assert.Contains(err.Error(), "nor from OS")
+		if err != nil {
+			assert.Contains(err.Error(), "nor from OS")
+		}
```

Comment:
Tiny nit - not a blocker though
```suggestion
		if !assert.Nil(err) {
		    return
		}
		
		assert.Contains(err.Error(), "nor from OS")
```